src/handlers/feature_handlers.cpp

```diff
@@ -6534,6 +6534,8 @@ void aiVehicleInstanceFeatureHandler::DrawHeadlights()
             lvlSegment segment;
             lvlIntersection intersection;
 
+            light->Position.AddScaled(vec, 0.f);
```

Comment:
Scaling something by 0 just gives 0, so this basically just a no-nop.

Reply:
Do you think I don't know that?! Because I wanna replace the code line above which has 0.1 multiplier if this option is enabled, so basically the game will use it if the option is disabled and the title of this commit explains that already

Reply:
> Do you think I don't know that?! Because I wanna replace the code line above which has 0.1 multiplier if this option is enabled, so basically the game will use it if the option is disabled and the title of this commit explains that already

@Wanted9811 Your explanation is a bit confusing, and there are no comments throughout your code, so how do we know what has been changed?

Reply:
I added comments to my latest code lines which explain what I did, I hope we're cool now.